client/asset/btc/btc.go

```diff
@@ -1083,6 +1080,9 @@ func (btc *ExchangeWallet) Unlock(pw string) error {
 
 // Lock locks the ExchangeWallet and the underlying bitcoind wallet.
 func (btc *ExchangeWallet) Lock() error {
+	if btc.node.Disconnected() {
+		return asset.ErrConnectionDown
+	}
```

Comment:
From what I can tell, since we're using `HTTPPostMode`, `Disconnected` is always `false`, (ultimately because `rpcclient.Client.disconnected` will never  be `true`).